docs/cheatsheet.rst

```diff
@@ -56,6 +56,17 @@ Members of ``address``
   returns ``false`` on failure
 - ``<address payable>.transfer(uint256 amount)``: send given amount of Wei to :ref:`address`, throws on failure
 
+.. warning::
+    `send()` and `transfer()` are deprecated and scheduled for removal.
```

Comment:
This warning should be in `units-and-global-variables.rst`. That's the canonical place where these things are fully documented. Cheatsheet is meant to be just a quick summary.

Reply:
Actually, "members of address" section in `units-and-global-variables.rst` seems redundant to me because we have a more detailed one already in [the `address` docs](https://docs.soliditylang.org/en/develop/types.html#members-of-addresses).

We should eventually merge the two. I would not do that now, but it would be good to add a link. And I would not repeat the full warning in both places. Docs of `address` should have a full one and the one here could just say that it's deprecated and direct the user to `address` docs for details.